Minor fixes + added details + bridging with SemWeb related works. (#25)
boucadair/rfc3535-20years-later: draft-boucadair-nmop-rfc3535-20years-later.md

## YANG-formatted Data Manipulation

The use of a flat tree hierarchy in YANG models may induce some performance issues compared to other graph models.
This can be the case, for example, during a path calculation on a network topology.

> Different approaches using graph paradigms and compatible with YANG are currently available, but require further experimentation to generalize their adoption.

Reviewer: ```suggestion
Different approaches using graph theory and compatible with YANG are currently available, but require further experimentation to generalize their adoption.
```

For istance, {{ODL}} implements an in-memory connected graph version of YANG-based data to enable fast breadth-first search (BFS).

> By combining the network digital twin concept {{?I-D.irtf-nmrg-network-digital-twin-arch}} with Semantic Web techniques, {{?I-D.marcas-nmop-knowledge-graph-yang}} and {{?I-D.tailhardat-nmop-incident-management-noria}} propose translating YANG-based data into the knowledge graph (KG) formalism to leverage automated reasoning and graph traversal techniques.

Reviewer: ```suggestion

NEW-OPS-REQ-Y2KG:
: Need for a reference specification to translate YANG-based data into the knowledge graph (KG).

For example, {{?I-D.marcas-nmop-knowledge-graph-yang}} and {{?I-D.tailhardat-nmop-incident-management-noria}} discuss YANG-2-KG proposals to leverage automated reasoning and graph traversal techniques.
```

Such convergence is also reflected on the need of interacting and interworking with distinct network parts participating in the end-to-end service delivery. Mobile access, fixed access, data center, enterprise, radio functional split (i.e., fronthaul and midhaul), neutral exchanges, intensive data networks (e.g., scientific academic networks), content distribution, etc., represent network parts constituent of end-to-end services that can impose dependencies of the management of an intermediate network.

That convergence observed in recent years also implies the need for an up-to-date refresh of management capabilities and tools for conventional networks.
It highlights the necessity to handle the heterogeneity of data, configuration, and network management/requirements.

> From a YANG perspective, this involves easily mapping and relating the data models used to manage each specific segment.

Reviewer: ```suggestion

From a YANG perspective, this involves easily mapping and relating the data models used to manage each specific segment.
```

> Resolving this kind of issue could draw on insights from parallel technical fields such as knowledge engineering practices and concepts associated with Linked Data in the Semantic Web, areas where it is common to manage problems of heterogeneity and data reconciliation across various application domains (e.g. {{?I-D.marcas-nmop-knowledge-graph-yang}} and {{?I-D.tailhardat-nmop-incident-management-noria}}).

Reviewer: ```suggestion
Resolving such issue could draw on insights from parallel technical fields such as knowledge engineering practices and concepts associated with Linked Data in the Semantic Web, areas where it is common to manage problems of heterogeneity and data reconciliation across various application domains.
```
